src/Client/MessageClient.php

```diff
@@ -68,11 +68,11 @@ public function buildCurlPostFields(Notification $notification): array
         }
 
         if (null !== $notification->getMessage()->getPriority()) {
-            $curlPostFields['priority'] = $notification->getMessage()->getPriority()->getPriority();
+            $curlPostFields['priority'] = (string) $notification->getMessage()->getPriority()->getPriority();
```

Comment:
Is there any particular reason for casting to `string` here? As far as I understand, `CURLOPT_POSTFIELDS` can contain `int`s as well as `string`s.

Reply:
It will create a URL query string internally, which does the casting internally otherwise. Better to be explicit here.

Reply:
If it is being taken care of internally anyways, then I think we dont have to be explicit here.

Reply:
Done

Reply:
Thank you so much!